tutorials/math/tStudent.py

```diff
@@ -0,0 +1,80 @@
+## \file
+## \ingroup tutorial_math
+## \notebook
+## Example macro describing the student t distribution
+##
+## ~~~{.cpp}
+## root[0]: .x tStudent.C
+## ~~~
+##
+## It draws the pdf, the cdf and then 10 quantiles of the t Student distribution
+##
+## based on Magdalena Slawinska's tStudent.C
+##
+## \macro_image
+## \macro_code
+##
+## \author Juan Fernando Jaramillo Botero
+
+from ROOT import TH1D, TF1, TCanvas, kRed, kBlue
+import ROOT
+
+
+# this is the way to force load of MathMore in Cling
+ROOT.Math.MathMoreLibrary.Load()
```

Comment:
Is this really needed or autoloading kicks in automatically?

Reply:
The line:
quant.Fill((i-0.5)/10.0, ROOT.Math.tdistribution_quantile((1.0 * i) / 10, 3.0))
requires this Load.